homeassistant/config.py

```diff
@@ -66,48 +66,15 @@
 )  # type: Tuple[Tuple[str, Any, Any, Optional[str]], ...]
 DEFAULT_CONFIG = """
 # Show links to resources in log and frontend
-introduction:
-
-# Enables the frontend
-frontend:
+default_config:
```

Comment:
Maybe a small warning. This should only removed from someone they known what they does.